metricbeat/module/php_fpm/process/_meta/data.json

```diff
@@ -12,26 +28,25 @@
         },
         "process": {
             "last_request_cpu": 0,
-            "last_request_memory": 2097152,
-            "request_duration": 1404,
```

Comment:
I think this was the request duration of the request processed by this process, that in this case happens to be the request to the status endpoint, but it could be any other thing.

Reply:
I will revert this change as the two durations are different: https://github.com/elastic/beats/pull/10366/files#r251730563